### PR TITLE
chore: support `latest-version` url format

### PR DESCRIPTION
**Pull request by mcdurdin**
This aligns with the work on kmn.sh for error message redirects from kmc.

See also keymanapp/keyman#10888.

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -53,83 +53,92 @@ RewriteRule "^keyboard/(?!redir.php)([^/?]+)/([^/?]+)(/)?$" "keyboard/redir.php?
...
+
+# latest-version:18.0
+# alpha or beta version reference
+RewriteCond "%{DOCUMENT_ROOT}/$1/18.0" -d
+RewriteRule "^(.+)/latest-version(/.*)?$" "$1/18.0$2" [DPI]
+
+# latest-version:17.0
+# alpha or beta version reference
+RewriteCond "%{DOCUMENT_ROOT}/$1/17.0" -d
+RewriteRule "^(.+)/latest-version(/.*)?$" "$1/17.0$2" [DPI]
```
Comment (darcywong00) [inline .htaccess:75]: What's the distinction between latest-version and current-version? Is current-version the current stable version?

Comment (mcdurdin) [reply .htaccess:75]: yes, `latest-version` is the most recent alpha release (e.g. 18.0 at this time) that can be found, whereas `current-version` is always going to be latest stable

Comment (mcdurdin) [reply .htaccess:75]: probably should have gone with `alpha` and `stable` as tags, still could do that moving forward!

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**